doc/contributing.md

```diff
@@ -110,102 +269,333 @@ Contributor information is optional but recommended.
+automated file checking, processing and ID harmonizations before 
+data is accepted for storage in _cimr-d_. One may check the status 
+of the automated steps by the colored dots next to the PR commit 
+history in the GitHub. Orange means the PR has been submitted and 
+is pending processing. Green means the PR has passed all 
+pre-requisites to proceed into the _cimr-d_ AWS S3 bucket for 
+public downloads.
+
+
+## I got an error message in the cimr log, what does it mean exactly?
+
+
+Troubleshooting cimr-d processing based on error messages:
+
+
+`data type is not recognized`
+* Currently _cimr-d_ expects variant-based association data. 
+  These can be genome-wide association study (gwas) results or 
+  expression-, splicing-, protein-, and other quantitative trait 
+  loci (eqtl, sqtl, pqtl, etc.). These data types should be 
+  indicated by the `data_type` field in the 
+  [yaml file data_info section](#data_info).
+
+
+`%s rows in %s are non-numeric' % (numcol, col,)`
+* variant_pos, inc_Afrq, effect_size, standard_error, zscore, 
+  pvalues, frequency, ma_samples, maf and tss_distance columns 
+  are expected to only contain numeric values.
+* By default the following values are interpreted as NaN: ‘’, 
+  ‘#N/A’, ‘#N/A N/A’, ‘#NA’, ‘-1.#IND’, ‘-1.#QNAN’, ‘-NaN’, 
+  ‘-nan’, ‘1.#IND’, ‘1.#QNAN’, ‘N/A’, ‘NA’, ‘NULL’, ‘NaN’, 
+  ‘n/a’, ‘nan’, ‘null’.
+
+
+`the format of %s is not testable.' % (col,)`
+* cimr will try to test whether above mentioned numeric columns 
+  containing non-numeric values can be converted into numeric. 
+  If this test fails, it will cause a format error.
+
+
+`unknown delimiter used in variant_id`
+* Expected variant_id format is:
+  chromosome_genomicposition_referenceallele_alternateallele_genomebuild
+  with underscores. cimr-d will accept ":" and "-" delimiters and 
+  convert them to underscores while processing. Any other delimiters or 
+  missing information may cause an error.
+
+
+`chromosome id needs to be checked.`
+* chromosome ID contains values other than \[chr\]1-26, X, Y, M or MT.
+* data is too big to be processed as a whole. Split chunks of data
+  do not contain all chromosomes (benign)
+
+
+`there are no matching rs ids`
+* By default a random subset of variants are selected to check against 
+  the reference genomic position - rs id pairs. If this test fails, 
+  cimr-d will cause an error.
+
+
+`{col} should only contain values between 0 and 1`
+* pvalue column containing non-probability values will cause an error.
+
+
+`feature_id column is not provided`
+* eqtl, sqtl, pqtl, etc. datasets must contain a column specifying the 
+  tested feature for each variant.
+
+
+`variant_id column is not provided`
+* variant_id column should contain unique variant ids in the format 
+  of chrom_pos_refallele_altallele_genomebuild.
+  e.g. chr1_13417_C_CGAGA_b37
+
+
+`rsnum column is not provided`
+* rsnum columns are recommended but _cimr_ will still run as long as 
```

Comment:
The previous line uses `rsnum column` (singular), but this line uses `rsnum columns` (plural), should we make it consistent?